### PR TITLE
Designed planhunt layout

### PR DESCRIPTION
**Pull request by Zephyr75**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/healthplay/planthunt/PlanthuntCreateLobbyActivity.java [hunk 1]
```diff
@@ -27,10 +25,10 @@ protected void onCreate(Bundle savedInstanceState) {
...
 
         //Get name and password fields in the activity
-        EditText editName = findViewById(R.id.createLobbyName);
-        EditText editPassword = findViewById(R.id.createLobbyPassword);
-        EditText editUsername = findViewById(R.id.createLobbyUsername);
-        lobbyButton = findViewById(R.id.createLobbyButton);
+        EditText editName = findViewById(R.id.planthuntCreateLobbyName);
+        EditText editPassword = findViewById(R.id.planthuntCreateLobbyPassword);
+        EditText editUsername = findViewById(R.id.planthuntCreateLobbyUsername);
+        lobbyButton = findViewById(R.id.planthuntCreateLobbyButton);
```
Comment (Tetardfurieux) [inline app/src/main/java/ch/epfl/sdp/healthplay/planthunt/PlanthuntCreateLobbyActivity.java:32]: Good choice of explicit name

**Review by Tetardfurieux — APPROVED** [review]
Not much to say since it's a lot of xml.
Anyway good code contribution and features !

**Review by ozan-guven — CHANGES_REQUESTED** [review]
I don't have much to say either but you can look for my one single comment. Otherwise good job!

**Review by ozan-guven — APPROVED** [review]
Good job !

**Pull request merged**